corehq/apps/app_manager/static/app_manager/js/details/case_detail_print.js

```diff
@@ -7,10 +7,8 @@ hqDefine("app_manager/js/details/case_detail_print", function () {
         printTemplateUploader = new HQMediaFileUploadController(
             print_uploader.slug,
             print_uploader.media_type,
-            _.extend({}, print_uploader.options, {
-                swfURL: initial_page_data('swfURL'),
-                sessionid: initial_page_data('sessionid'),
-            }));
+            print_uploader.options,
+        );
```

Comment:
What am I missing. Our [.eslintrc.js](https://github.com/dimagi/commcare-hq/blob/master/.eslintrc.js#L39) requires `comma-dangle`, and Parsing error isn't what I'd expect in that situation anyway, so it can't be that. Am I like counting parentheses wrong?

Reply:
comma-dangle is for objects & arrays, but not parameter lists in function calls. This will also break deploy because the uglifier will interpret it as a syntax error.

Today I learned that the trailing comma *is* allowed in function calls as of ECMAScript, but we're not there yet.

Reply:
Ah, okay thank you sharing that. I assumed because of https://eslint.org/docs/rules/comma-dangle#functions that `comma-dangle` included function arguments. But maybe this parsing error is coming from something other than eslint 🤔

Reply:
I think eslint has logic so that function arguments aren't included in the `comma-dangle` rule, even when it's set to "always", unless you also have an eslint environment that supports that syntax, like by setting `ecmaVersion` to 8+

Reply:
Ah okay gotcha. That was the piece I was missing. Thanks!